demosplan/DemosPlanCoreBundle/Logic/Statement/AssessmentHandler.php

```diff
@@ -88,6 +88,11 @@ public function __construct(
         $this->userFilterSetService = $userFilterSetService;
     }
 
+    public function getStatementService(): StatementService
```

Comment:
nope, don't do this!

Reply:
adjusted in [795ea62](https://github.com/demos-europe/demosplan-core/pull/2337/commits/795ea628adb9fb5da1af0e72ea3b94f480bbf36b)